diff --git a/xsd-frontend/semantic-graph/complex.hxx b/xsd-frontend/semantic-graph/complex.hxx
index ac47810..107f5f4 100644
--- a/xsd-frontend/semantic-graph/complex.hxx
+++ b/xsd-frontend/semantic-graph/complex.hxx
@@ -15,6 +15,11 @@ namespace XSDFrontend
     class Complex: public virtual Type, public virtual Scope
     {
     public:
+      bool
+      abstract_p () const {
+        return false;
+      }
+      
       bool
       mixed_p () const
       {
