Subject: [PATCH] init
---
Index: xsd/cxx/parser/state-processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/state-processor.cxx b/xsd/cxx/parser/state-processor.cxx
--- a/xsd/cxx/parser/state-processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/state-processor.cxx	(date 1763240555210)
@@ -8,8 +8,8 @@
 
 #include <xsd/cxx/parser/elements.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/parser/print-impl-common.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/print-impl-common.hxx b/xsd/cxx/parser/print-impl-common.hxx
--- a/xsd/cxx/parser/print-impl-common.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/print-impl-common.hxx	(date 1763240555203)
@@ -4,8 +4,8 @@
 #ifndef XSD_CXX_PARSER_PRINT_IMPL_COMMON_HXX
 #define XSD_CXX_PARSER_PRINT_IMPL_COMMON_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/parser/elements.hxx>
 
Index: xsd/cxx/parser/name-processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/name-processor.cxx b/xsd/cxx/parser/name-processor.cxx
--- a/xsd/cxx/parser/name-processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/name-processor.cxx	(date 1763240555149)
@@ -8,8 +8,8 @@
 
 #include <xsd/cxx/parser/name-processor.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 
 using namespace std;
Index: xsd/cxx/parser/name-processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/name-processor.hxx b/xsd/cxx/parser/name-processor.hxx
--- a/xsd/cxx/parser/name-processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/name-processor.hxx	(date 1763240555220)
@@ -4,7 +4,7 @@
 #ifndef XSD_CXX_PARSER_NAME_PROCESSOR_HXX
 #define XSD_CXX_PARSER_NAME_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
 
 #include <xsd/types.hxx>
 
Index: xsd/cxx/parser/parser-inline.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/parser-inline.cxx b/xsd/cxx/parser/parser-inline.cxx
--- a/xsd/cxx/parser/parser-inline.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/parser-inline.cxx	(date 1763240555221)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/parser/parser-inline.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/parser-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/parser-header.cxx b/xsd/cxx/parser/parser-header.cxx
--- a/xsd/cxx/parser/parser-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/parser-header.cxx	(date 1763240555209)
@@ -5,8 +5,8 @@
 
 #include <xsd/cxx/parser/parser-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/parser/impl-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/impl-source.cxx b/xsd/cxx/parser/impl-source.cxx
--- a/xsd/cxx/parser/impl-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/impl-source.cxx	(date 1763240555182)
@@ -4,8 +4,8 @@
 #include <xsd/cxx/parser/impl-source.hxx>
 #include <xsd/cxx/parser/print-impl-common.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/characters-validation-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/characters-validation-source.cxx b/xsd/cxx/parser/characters-validation-source.cxx
--- a/xsd/cxx/parser/characters-validation-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/characters-validation-source.cxx	(date 1763240555167)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/parser/characters-validation-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/parser-forward.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/parser-forward.cxx b/xsd/cxx/parser/parser-forward.cxx
--- a/xsd/cxx/parser/parser-forward.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/parser-forward.cxx	(date 1763240555183)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/parser/parser-forward.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/element-validation-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/element-validation-source.cxx b/xsd/cxx/parser/element-validation-source.cxx
--- a/xsd/cxx/parser/element-validation-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/element-validation-source.cxx	(date 1763240555145)
@@ -5,8 +5,8 @@
 
 #include <xsd/cxx/parser/element-validation-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/parser/state-processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/state-processor.hxx b/xsd/cxx/parser/state-processor.hxx
--- a/xsd/cxx/parser/state-processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/state-processor.hxx	(date 1763240555154)
@@ -4,7 +4,7 @@
 #ifndef XSD_CXX_PARSER_STATE_PROCESSOR_HXX
 #define XSD_CXX_PARSER_STATE_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
 
 #include <xsd/types.hxx>
 
Index: xsd/cxx/parser/driver-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/driver-source.cxx b/xsd/cxx/parser/driver-source.cxx
--- a/xsd/cxx/parser/driver-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/driver-source.cxx	(date 1763240555157)
@@ -8,8 +8,8 @@
 #include <xsd/cxx/parser/driver-source.hxx>
 #include <xsd/cxx/parser/print-impl-common.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/parser/parser-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/parser-source.cxx b/xsd/cxx/parser/parser-source.cxx
--- a/xsd/cxx/parser/parser-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/parser-source.cxx	(date 1763240555198)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/parser/parser-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/type-processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/type-processor.hxx b/xsd/cxx/parser/type-processor.hxx
--- a/xsd/cxx/parser/type-processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/type-processor.hxx	(date 1763240555224)
@@ -4,7 +4,7 @@
 #ifndef XSD_CXX_PARSER_TYPE_PROCESSOR_HXX
 #define XSD_CXX_PARSER_TYPE_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
 
 #include <xsd/types.hxx>
 
Index: xsd/cxx/parser/generator.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/generator.cxx b/xsd/cxx/parser/generator.cxx
--- a/xsd/cxx/parser/generator.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/generator.cxx	(date 1763240555226)
@@ -11,7 +11,7 @@
 #include <cutl/compiler/cxx-indenter.hxx>
 #include <cutl/compiler/sloc-counter.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
 
 #include <xsd/version.hxx>
 
Index: xsd/cxx/parser/impl-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/impl-header.cxx b/xsd/cxx/parser/impl-header.cxx
--- a/xsd/cxx/parser/impl-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/impl-header.cxx	(date 1763240555207)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/parser/impl-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/generator.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/generator.hxx b/xsd/cxx/parser/generator.hxx
--- a/xsd/cxx/parser/generator.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/generator.hxx	(date 1763240555189)
@@ -4,8 +4,8 @@
 #ifndef XSD_CXX_PARSER_GENERATOR_HXX
 #define XSD_CXX_PARSER_GENERATOR_HXX
 
-#include <libxsd-frontend/semantic-graph/elements.hxx> // Path
-#include <libxsd-frontend/semantic-graph/schema.hxx>
+#include <xsd-frontend/semantic-graph/elements.hxx> // Path
+#include <xsd-frontend/semantic-graph/schema.hxx>
 
 #include <xsd/xsd.hxx>
 #include <xsd/types.hxx>
Index: xsd/cxx/parser/type-processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/type-processor.cxx b/xsd/cxx/parser/type-processor.cxx
--- a/xsd/cxx/parser/type-processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/type-processor.cxx	(date 1763240555156)
@@ -6,8 +6,8 @@
 #include <xsd/cxx/parser/elements.hxx>
 #include <xsd/cxx/parser/type-processor.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/parser/attribute-validation-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/attribute-validation-source.cxx b/xsd/cxx/parser/attribute-validation-source.cxx
--- a/xsd/cxx/parser/attribute-validation-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/attribute-validation-source.cxx	(date 1763240555212)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/parser/attribute-validation-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/parser/validator.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/parser/validator.cxx b/xsd/cxx/parser/validator.cxx
--- a/xsd/cxx/parser/validator.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/parser/validator.cxx	(date 1763240555151)
@@ -6,8 +6,8 @@
 
 #include <xsd/cxx/parser/validator.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/parser/elements.hxx>
 
Index: xsd/processing/inheritance/processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/processing/inheritance/processor.cxx b/xsd/processing/inheritance/processor.cxx
--- a/xsd/processing/inheritance/processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/processing/inheritance/processor.cxx	(date 1763240555223)
@@ -8,8 +8,8 @@
 
 #include <xsd/elements.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/processing/cardinality/processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/processing/cardinality/processor.hxx b/xsd/processing/cardinality/processor.hxx
--- a/xsd/processing/cardinality/processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/processing/cardinality/processor.hxx	(date 1763240555186)
@@ -4,8 +4,8 @@
 #ifndef XSD_PROCESSING_CARDINALITY_PROCESSOR_HXX
 #define XSD_PROCESSING_CARDINALITY_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph/elements.hxx> // Path
-#include <libxsd-frontend/semantic-graph/schema.hxx>
+#include <xsd-frontend/semantic-graph/elements.hxx> // Path
+#include <xsd-frontend/semantic-graph/schema.hxx>
 
 #include <xsd/types.hxx>
 
Index: xsd/processing/inheritance/processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/processing/inheritance/processor.hxx b/xsd/processing/inheritance/processor.hxx
--- a/xsd/processing/inheritance/processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/processing/inheritance/processor.hxx	(date 1763240555188)
@@ -4,8 +4,8 @@
 #ifndef XSD_PROCESSING_INHERITANCE_PROCESSOR_HXX
 #define XSD_PROCESSING_INHERITANCE_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph/elements.hxx> // Path
-#include <libxsd-frontend/semantic-graph/schema.hxx>
+#include <xsd-frontend/semantic-graph/elements.hxx> // Path
+#include <xsd-frontend/semantic-graph/schema.hxx>
 
 #include <xsd/types.hxx>
 
Index: xsd/processing/cardinality/processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/processing/cardinality/processor.cxx b/xsd/processing/cardinality/processor.cxx
--- a/xsd/processing/cardinality/processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/processing/cardinality/processor.cxx	(date 1763240555179)
@@ -3,8 +3,8 @@
 
 #include <map>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/elements.hxx>
 
Index: xsd/version.hxx.in
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/version.hxx.in b/xsd/version.hxx.in
--- a/xsd/version.hxx.in	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/version.hxx.in	(date 1763240555164)
@@ -48,7 +48,7 @@
 
 $libcutl.check(LIBCUTL_VERSION, LIBCUTL_SNAPSHOT)$
 
-#include <libxsd-frontend/version.hxx>
+#include <xsd-frontend/version.hxx>
 
 $libxsd_frontend.check(LIBXSD_FRONTEND_VERSION, LIBXSD_FRONTEND_SNAPSHOT)$
 
Index: xsd/xsd.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/xsd.hxx b/xsd/xsd.hxx
--- a/xsd/xsd.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/xsd.hxx	(date 1763240555172)
@@ -11,7 +11,7 @@
 #include <cutl/shared-ptr.hxx>
 #include <cutl/fs/auto-remove.hxx>
 
-#include <libxsd-frontend/semantic-graph/elements.hxx> // Path
+#include <xsd-frontend/semantic-graph/elements.hxx> // Path
 
 #include <xsd/types.hxx>
 
Index: xsd/xsd.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/xsd.cxx b/xsd/xsd.cxx
--- a/xsd/xsd.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/xsd.cxx	(date 1763240555218)
@@ -12,12 +12,12 @@
 
 #include <cutl/re.hxx>
 
-#include <libxsd-frontend/parser.hxx>
-#include <libxsd-frontend/transformations/anonymous.hxx>
-#include <libxsd-frontend/transformations/enum-synthesis.hxx>
-#include <libxsd-frontend/transformations/restriction.hxx>
-#include <libxsd-frontend/transformations/schema-per-type.hxx>
-#include <libxsd-frontend/transformations/simplifier.hxx>
+#include <xsd-frontend/parser.hxx>
+#include <xsd-frontend/transformations/anonymous.hxx>
+#include <xsd-frontend/transformations/enum-synthesis.hxx>
+#include <xsd-frontend/transformations/restriction.hxx>
+#include <xsd-frontend/transformations/schema-per-type.hxx>
+#include <xsd-frontend/transformations/simplifier.hxx>
 
 #include <xsd/version.hxx>
 
Index: xsd/types.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/types.hxx b/xsd/types.hxx
--- a/xsd/types.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/types.hxx	(date 1763240555195)
@@ -7,7 +7,7 @@
 #include <vector>
 #include <cstddef> // std::size_t
 
-#include <libxsd-frontend/types.hxx>
+#include <xsd-frontend/types.hxx>
 
 using std::size_t;
 
Index: xsd/elements.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/elements.hxx b/xsd/elements.hxx
--- a/xsd/elements.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/elements.hxx	(date 1763240555140)
@@ -4,8 +4,8 @@
 #ifndef XSD_ELEMENTS_HXX
 #define XSD_ELEMENTS_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/types.hxx>
 
Index: xsd/cxx/tree/order-processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/order-processor.cxx b/xsd/cxx/tree/order-processor.cxx
--- a/xsd/cxx/tree/order-processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/order-processor.cxx	(date 1763240555228)
@@ -6,8 +6,8 @@
 #include <xsd/cxx/tree/elements.hxx>
 #include <xsd/cxx/tree/order-processor.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/tree/fundamental-header.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/fundamental-header.hxx b/xsd/cxx/tree/fundamental-header.hxx
--- a/xsd/cxx/tree/fundamental-header.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/fundamental-header.hxx	(date 1763240555193)
@@ -8,8 +8,8 @@
 #include <vector>
 #include <algorithm>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/tree/elements.hxx>
 
Index: xsd/cxx/tree/order-processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/order-processor.hxx b/xsd/cxx/tree/order-processor.hxx
--- a/xsd/cxx/tree/order-processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/order-processor.hxx	(date 1763240555152)
@@ -4,7 +4,7 @@
 #ifndef XSD_CXX_TREE_ORDER_PROCESSOR_HXX
 #define XSD_CXX_TREE_ORDER_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
 
 #include <xsd/xsd.hxx>
 #include <xsd/types.hxx>
Index: xsd/cxx/literal-map.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/literal-map.cxx b/xsd/cxx/literal-map.cxx
--- a/xsd/cxx/literal-map.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/literal-map.cxx	(date 1763240555166)
@@ -16,7 +16,7 @@
 #include <xercesc/sax2/SAX2XMLReader.hpp>
 #include <xercesc/sax2/XMLReaderFactory.hpp>
 
-#include <libxsd-frontend/xml.hxx>
+#include <xsd-frontend/xml.hxx>
 
 #include <xsd/cxx/literal-map.hxx>
 
Index: xsd/cxx/elements.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/elements.hxx b/xsd/cxx/elements.hxx
--- a/xsd/cxx/elements.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/elements.hxx	(date 1763240555231)
@@ -11,8 +11,8 @@
 
 #include <cutl/re.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/types.hxx>
 #include <xsd/elements.hxx>
Index: xsd/cxx/tree/parser-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/parser-header.cxx b/xsd/cxx/tree/parser-header.cxx
--- a/xsd/cxx/tree/parser-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/parser-header.cxx	(date 1763240555196)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/parser-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/tree-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/tree-source.cxx b/xsd/cxx/tree/tree-source.cxx
--- a/xsd/cxx/tree/tree-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/tree-source.cxx	(date 1763240555205)
@@ -4,8 +4,8 @@
 #include <map>
 #include <list>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/tree/tree-source.hxx>
 #include <xsd/cxx/tree/default-value.hxx>
Index: xsd/cxx/tree/stream-insertion-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/stream-insertion-header.cxx b/xsd/cxx/tree/stream-insertion-header.cxx
--- a/xsd/cxx/tree/stream-insertion-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/stream-insertion-header.cxx	(date 1763240555216)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/stream-insertion-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/stream-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/stream-header.cxx b/xsd/cxx/tree/stream-header.cxx
--- a/xsd/cxx/tree/stream-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/stream-header.cxx	(date 1763240555180)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/stream-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/default-value.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/default-value.hxx b/xsd/cxx/tree/default-value.hxx
--- a/xsd/cxx/tree/default-value.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/default-value.hxx	(date 1763240555201)
@@ -4,8 +4,8 @@
 #ifndef XSD_CXX_TREE_DEFAULT_VALUE_HXX
 #define XSD_CXX_TREE_DEFAULT_VALUE_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/tree/elements.hxx>
 
Index: xsd/cxx/tree/stream-extraction-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/stream-extraction-source.cxx b/xsd/cxx/tree/stream-extraction-source.cxx
--- a/xsd/cxx/tree/stream-extraction-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/stream-extraction-source.cxx	(date 1763240555175)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/stream-extraction-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/tree-inline.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/tree-inline.cxx b/xsd/cxx/tree/tree-inline.cxx
--- a/xsd/cxx/tree/tree-inline.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/tree-inline.cxx	(date 1763240555200)
@@ -1,8 +1,8 @@
 // file      : xsd/cxx/tree/tree-inline.cxx
 // license   : GNU GPL v2 + exceptions; see accompanying LICENSE file
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/tree/tree-inline.hxx>
 #include <xsd/cxx/tree/default-value.hxx>
Index: xsd/cxx/tree/tree-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/tree-header.cxx b/xsd/cxx/tree/tree-header.cxx
--- a/xsd/cxx/tree/tree-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/tree-header.cxx	(date 1763240555170)
@@ -1,8 +1,8 @@
 // file      : xsd/cxx/tree/tree-header.cxx
 // license   : GNU GPL v2 + exceptions; see accompanying LICENSE file
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/tree/tree-header.hxx>
 #include <xsd/cxx/tree/default-value.hxx>
Index: xsd/cxx/tree/parser-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/parser-source.cxx b/xsd/cxx/tree/parser-source.cxx
--- a/xsd/cxx/tree/parser-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/parser-source.cxx	(date 1763240555233)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/parser-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/serialization-header.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/serialization-header.cxx b/xsd/cxx/tree/serialization-header.cxx
--- a/xsd/cxx/tree/serialization-header.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/serialization-header.cxx	(date 1763240555147)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/serialization-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/polymorphism-processor.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/polymorphism-processor.hxx b/xsd/cxx/tree/polymorphism-processor.hxx
--- a/xsd/cxx/tree/polymorphism-processor.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/polymorphism-processor.hxx	(date 1763240555185)
@@ -4,7 +4,7 @@
 #ifndef XSD_CXX_TREE_POLYMORPHISM_PROCESSOR_HXX
 #define XSD_CXX_TREE_POLYMORPHISM_PROCESSOR_HXX
 
-#include <libxsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
 
 #include <xsd/xsd.hxx>
 #include <xsd/types.hxx>
Index: xsd/cxx/tree/validator.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/validator.cxx b/xsd/cxx/tree/validator.cxx
--- a/xsd/cxx/tree/validator.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/validator.cxx	(date 1763240555174)
@@ -6,8 +6,8 @@
 
 #include <xsd/cxx/tree/validator.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 #include <xsd/cxx/tree/elements.hxx>
 
Index: xsd/cxx/tree/serialization-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/serialization-source.cxx b/xsd/cxx/tree/serialization-source.cxx
--- a/xsd/cxx/tree/serialization-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/serialization-source.cxx	(date 1763240555163)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/serialization-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/stream-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/stream-source.cxx b/xsd/cxx/tree/stream-source.cxx
--- a/xsd/cxx/tree/stream-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/stream-source.cxx	(date 1763240555229)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/stream-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/stream-insertion-source.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/stream-insertion-source.cxx b/xsd/cxx/tree/stream-insertion-source.cxx
--- a/xsd/cxx/tree/stream-insertion-source.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/stream-insertion-source.cxx	(date 1763240555177)
@@ -3,8 +3,8 @@
 
 #include <xsd/cxx/tree/stream-insertion-source.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/generator.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/generator.hxx b/xsd/cxx/tree/generator.hxx
--- a/xsd/cxx/tree/generator.hxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/generator.hxx	(date 1763240555213)
@@ -4,8 +4,8 @@
 #ifndef XSD_CXX_TREE_GENERATOR_HXX
 #define XSD_CXX_TREE_GENERATOR_HXX
 
-#include <libxsd-frontend/semantic-graph/elements.hxx> // Path
-#include <libxsd-frontend/semantic-graph/schema.hxx>
+#include <xsd-frontend/semantic-graph/elements.hxx> // Path
+#include <xsd-frontend/semantic-graph/schema.hxx>
 
 #include <xsd/xsd.hxx>
 #include <xsd/types.hxx>
Index: xsd/cxx/tree/tree-forward.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/tree-forward.cxx b/xsd/cxx/tree/tree-forward.cxx
--- a/xsd/cxx/tree/tree-forward.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/tree-forward.cxx	(date 1763240555143)
@@ -4,8 +4,8 @@
 #include <xsd/cxx/tree/tree-forward.hxx>
 #include <xsd/cxx/tree/fundamental-header.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 namespace CXX
 {
Index: xsd/cxx/tree/polymorphism-processor.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/polymorphism-processor.cxx b/xsd/cxx/tree/polymorphism-processor.cxx
--- a/xsd/cxx/tree/polymorphism-processor.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/polymorphism-processor.cxx	(date 1763240555191)
@@ -6,8 +6,8 @@
 #include <xsd/cxx/tree/elements.hxx>
 #include <xsd/cxx/tree/polymorphism-processor.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/traversal.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/traversal.hxx>
 
 using namespace std;
 
Index: xsd/cxx/tree/generator.cxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/generator.cxx b/xsd/cxx/tree/generator.cxx
--- a/xsd/cxx/tree/generator.cxx	(revision 0f253f3c87a6b310e7b310bd3c883f131c74d2f6)
+++ b/xsd/cxx/tree/generator.cxx	(date 1763240555161)
@@ -13,8 +13,8 @@
 #include <cutl/compiler/cxx-indenter.hxx>
 #include <cutl/compiler/sloc-counter.hxx>
 
-#include <libxsd-frontend/semantic-graph.hxx>
-#include <libxsd-frontend/generators/dependencies.hxx>
+#include <xsd-frontend/semantic-graph.hxx>
+#include <xsd-frontend/generators/dependencies.hxx>
 
 #include <xsd/version.hxx>
 
