diff --git a/xsd/version.hxx.in b/xsd/version.hxx.in
index 8053e21..e501ebc 100644
--- a/xsd/version.hxx.in
+++ b/xsd/version.hxx.in
@@ -22,19 +22,19 @@
 // 3.0.0-b.2    0000299999999995020
 // 2.2.0-a.1.z  0000200001999990011
 //
-#define XSD_VERSION       $xsd.version.project_number$ULL
-#define XSD_VERSION_STR   "$xsd.version.project$"
-#define XSD_VERSION_ID    "$xsd.version.project_id$"
-#define XSD_VERSION_FULL  "$xsd.version$"
+#cmakedefine XSD_VERSION       @XSD_VERSION@
+#cmakedefine XSD_VERSION_STR   "@XSD_VERSION_STR@"
+#cmakedefine XSD_VERSION_ID    "@XSD_VERSION_ID@"
+#cmakedefine XSD_VERSION_FULL  "@XSD_VERSION_FULL@"
 
-#define XSD_VERSION_MAJOR $xsd.version.major$
-#define XSD_VERSION_MINOR $xsd.version.minor$
-#define XSD_VERSION_PATCH $xsd.version.patch$
+#cmakedefine XSD_VERSION_MAJOR @XSD_VERSION_MAJOR@
+#cmakedefine XSD_VERSION_MINOR @XSD_VERSION_MINOR@
+#cmakedefine XSD_VERSION_PATCH @XSD_VERSION_PATCH@
 
-#define XSD_PRE_RELEASE   $xsd.version.pre_release$
+#cmakedefine XSD_PRE_RELEASE   @XSD_PRE_RELEASE@
 
-#define XSD_SNAPSHOT      $xsd.version.snapshot_sn$ULL
-#define XSD_SNAPSHOT_ID   "$xsd.version.snapshot_id$"
+#cmakedefine XSD_SNAPSHOT      @XSD_SNAPSHOT@
+#cmakedefine XSD_SNAPSHOT_ID   "@XSD_SNAPSHOT_ID@"
 
 #include <xercesc/util/XercesVersion.hpp>
 
@@ -44,12 +44,4 @@
 #  error Xerces-C++ 2-series is not supported
 #endif
 
-#include <cutl/version.hxx>
-
-$libcutl.check(LIBCUTL_VERSION, LIBCUTL_SNAPSHOT)$
-
-#include <xsd-frontend/version.hxx>
-
-$libxsd_frontend.check(LIBXSD_FRONTEND_VERSION, LIBXSD_FRONTEND_SNAPSHOT)$
-
 #endif // XSD_VERSION
