Index: xsd/cxx/tree/elements.hxx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/xsd/cxx/tree/elements.hxx b/xsd/cxx/tree/elements.hxx
--- a/xsd/cxx/tree/elements.hxx	(revision 5a2a9e262250f5c8c0f9e38781958adf19b91e71)
+++ b/xsd/cxx/tree/elements.hxx	(date 1763244534581)
@@ -224,6 +224,11 @@
       bool
       polymorphic_p (SemanticGraph::Type&);
 
+      bool
+      abstract_p () {
+        return false;
+      }
+
       bool
       anonymous_p (SemanticGraph::Type const& t)
       {
